lib/ansible/cli/galaxy.py

```diff
@@ -1393,7 +1393,10 @@ def _execute_install_collection(
         upgrade = context.CLIARGS.get('upgrade', False)
 
         collections_path = C.COLLECTIONS_PATHS
-        if len([p for p in collections_path if p.startswith(path)]) == 0:
+        if (
+            C.GALAXY_COLLECTIONS_PATH_WARNING
+            and len([p for p in collections_path if p.startswith(path)]) == 0
```

Comment:
Do you mind if I change this to

```suggestion
            and not [p for p in collections_path if p.startswith(path)]
```
?

Reply:
That works for me :+1: